projects/akita-ng-fire/src/lib/collection-group/collection-group.service.ts

```diff
@@ -50,13 +59,14 @@ export abstract class CollectionGroupService<S extends EntityState> {
       setLoading(storeName, true);
     }
     return this.db.collectionGroup(this.collectionId, query).stateChanges().pipe(
-      withTransaction(actions => syncStoreFromDocAction(storeName, actions, this.idKey))
+      map(value => this.formatFromFirestore(value)),
```

Comment:
Have you tried it ? You're appling the format on the stateChange of list of document. It shouldn't work